src/http/api/follow.ts

```diff
@@ -107,11 +68,14 @@ export class FollowController {
 
         await apCtx.sendActivity(
             { username: followerAccount.username },
-            actorToFollow,
+            {
+                id: followedAccount.apId,
+                inboxId: followedAccount.apInbox,
+            },
             follow,
         );
 
-        return new Response(JSON.stringify(await actorToFollow.toJsonLd()), {
+        return new Response(JSON.stringify({}), {
```

Comment:
I think we should return the AccountDTO here

Reply:
I went against this - because 1. were not using the response and 2. it requires a bunch more queries